langkit/compiled_types.py

```diff
@@ -3649,8 +3647,22 @@ def __init__(self, name, location, doc, value_names):
         :type: dict[names.Name: EnumValue]
         """
 
+        self.default_val_name = default_val_name
+        """
+        Name of the default value for this enum, if any
+
+        :type: names.Name|None
+        """
+
         CompiledTypeRepo.enum_types.append(self)
 
+        super(EnumType, self).__init__(
+            name, location, doc, is_ptr=False, exposed=True,
+            null_allowed=default_val_name is not None,
+            nullexpr=self.values_dict[default_val_name].ada_name
+            if default_val_name is not None else None,
```

Comment:
Can you reindent the `if` so it’s clear that it applies to `self.values_dict[…`?